kubejs/server_scripts/enigmatica/kubejs/base/unification/unify_materials.js

```diff
@@ -627,6 +628,34 @@ onEvent('recipes', (event) => {
             .id(`immersiveengineering:crusher/ore_${material}`);
     }
 
+    function mekanism_ingot_gem_crushing(event, material, ingot, dust, gem) {
+        if (dust == air) {
```

Comment:
Please check ingot too :)

Reply:
It seems redundant to check for ingot and I would assume gem here aswell? The if & elif check below checks both of them to ensure they are not air, to set the proper input type. If it is air, it falls to the else statement anyways and returns nothing

Reply:
Oh, right ^^' you're completely right about that.